frontend/src/APIClients/ReviewAPIClient.ts

```diff
@@ -16,4 +16,25 @@ const getReviews = async (): Promise<ReviewResponse[]> => {
   }
 };
 
-export default { getReviews };
+/**
+ * Publishes a review
+ * @param review - The review to publish
+ * @returns The published review
+ */
+const publishReview = async (
+  review: ReviewRequest,
+): Promise<ReviewResponse | null> => {
+  try {
+    const { data } = await baseAPIClient.post("/reviews", review, {
+      headers: {
+        Authorization: BEARER_TOKEN,
+        "Content-Type": "application/json",
+      },
+    });
+    return data;
```

Comment:
Do you know the type of this return? Is it just JSON or a response DTO?

Reply:
type of the return is given in the component header? 

`const publishReview = async (
  review: ReviewRequest,
): Promise<ReviewResponse | null>` 

data should be `ReviewResponse`